futhark_ffi/__init__.py

```diff
@@ -157,3 +160,12 @@ def wrapper(*args):
                 return tuple(results)
 
         return wrapper
+
+    def pause_profiling(self):
+        self.lib.futhark_context_pause_profiling(self.ctx)
+
+    def unpause_profiling(self):
+        self.lib.futhark_context_unpause_profiling(self.ctx)
+
+    def report(self):
+        return self.ffi.string(self.lib.futhark_context_report(self.ctx)).decode()
```

Comment:
We should free the string returned here; quoth [the docs](https://futhark.readthedocs.io/en/latest/c-api.html#c.futhark_context_report): "It is the caller’s responsibility to free the returned string."

Reply:
Thanks, I missed that. I also added a helper method since this seems to be pretty common.